src/EnergyPlus/WeatherManager.cc

```diff
@@ -2334,8 +2338,13 @@ namespace WeatherManager {
         state.dataEnvrn->OpaqueCloudCover = state.dataWeatherManager->TodayOpaqueSkyCover(state.dataGlobal->TimeStep, state.dataGlobal->HourOfDay);
 
         if (state.dataWeatherManager->UseRainValues) {
-            state.dataEnvrn->IsRain = state.dataWeatherManager->TodayIsRain(
-                state.dataGlobal->TimeStep, state.dataGlobal->HourOfDay); //.or. LiquidPrecipitation >= .8d0)  ! > .8 mm
```

Comment:
Keep this code but add `if (state.dataWaterData->RainFall.ModeID == DataWater::RainfallMode::EPWPrecipitation)` and add a comment pointing to the code that sets `TomorrowIsRain` based on the 0.8mm/hr threshold.